Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -614,6 +615,14 @@ public void dispose() {
     return myServerData.getErrors(file);
   }
 
+  public List<DartServerData.DartError> getErrors(@NotNull final Module module) {
```

Comment:
Sounds good! A few of the public APIs are unused by this PR - they are more there for completeness, to make the API a bit more robust possible future client needs.